workingBranch

KosMas-prog opened the pull request.

KosMas-prog committed:
first commit

KosMas-prog committed:
Выполнение задания 1.1.3

MilenaG03 commented on src/main/java/jm/task/core/jdbc/Main.java:
```diff
@@ -1,7 +1,35 @@
+import java.sql.Connection;
+import java.sql.SQLException;
+
 public class Main {
```
папки .idea быть в реозитории не должно

KosMas-prog replied:
Видимо не правильно написал в гитигнор... (без слэшей). Вроде исправил...

MilenaG03 commented on src/main/java/jm/task/core/jdbc/Main.java:
```diff
@@ -1,7 +1,35 @@
+        dao.saveUser("Sidor", "Sidorov", (byte) 22);
+        dao.saveUser("Vasya", "Vasiliev", (byte) 23);
+        dao.getAllUsers();
+       // dao.cleanUsersTable();
```
зачем закоммеентировал? по задаче и удаление и очистка таблицы должны быть

KosMas-prog replied:
Забыл убрать слэши... Смотрел таблицу в базе данных...

MilenaG03 commented on src/main/java/jm/task/core/jdbc/dao/UserDaoJDBCImpl.java:
```diff
@@ -1,35 +1,90 @@
+import java.util.ArrayList;
 import java.util.List;
 
 public class UserDaoJDBCImpl implements UserDao {
```
подключи Lombok и используй во всех классах

KosMas-prog replied:
Спасибо за подсказку! Подключил. Изучаю...

MilenaG03 replied:
сюда тоже аннотацию повесь

MilenaG03 commented on src/main/java/jm/task/core/jdbc/util/Util.java:
```diff
@@ -1,5 +1,35 @@
+        try {
+            Class.forName("org.postgresql.Driver");
+            connection = DriverManager.getConnection(URL, USERNAME, PASSWORD);
+            System.out.println("Connection to Task1 DB succeeded!");
```
лучше подключи SLF4G и для таких вещей используй её

KosMas-prog committed:
1. Добавил папку .idea в гитигнор
2. Подключил SLF4J
3. Раскомментировал очистку и удаление таблицы
4. Lombok добавил (спасибо за подсказку)

KosMas-prog committed:
Remove files

KosMas-prog committed:
Удалил папку .idea из репозитория

KosMas-prog committed:
1. Добавил папку .idea в гитигнор
2. Подключил SLF4J
3. Раскомментировал очистку и удаление таблицы
4. Lombok добавил (спасибо за подсказку)

MilenaG03 commented on src/main/java/jm/task/core/jdbc/service/UserServiceImpl.java:
```diff
@@ -1,31 +1,38 @@
 
 import java.util.List;
 
 public class UserServiceImpl implements UserService {
```
вот сюда тожно можно аннотацию из ломбока добавить, подумай какую

MilenaG03 replied:
@KosMas-prog

KosMas-prog committed:
Добавил в класс UserServiceImpl аннотацию @RequiredArgsConstructor.
Добавил в класс UserDaoJDBCImpl аннотацию @Slf4j
